daemon.go

```diff
@@ -167,13 +192,11 @@ func build() bool {
 
 	output, err := cmd.CombinedOutput()
 
-	if err == nil {
-		log.Println(okColor("Build ok."))
-	} else {
+	if err != nil {
 		log.Println(failColor("Error while building:\n"), failColor(string(output)))
+		return false
 	}
-
-	return err == nil
+	return true
```

Comment:
I think I would have returned the error and did the logging one level higher up for better separation of concerns.

Reply:
I've added a success log message if a command was success. The executed command will also be logged in both case (error/success).
Now you can see in the output, which command has failed or was success.